### PR TITLE
sci-mathematics/nestedsums: Drop USE=static-libs

### PR DESCRIPTION
**Pull request by jsmolic**

### CODE REVIEW

File: sci-mathematics/nestedsums/nestedsums-1.5.2-r1.ebuild [hunk 1]
```diff
@@ -35,11 +37,12 @@ src_test() {
```
Comment (SoapGentoo) [inline sci-mathematics/nestedsums/nestedsums-1.5.2-r1.ebuild:37]: I think you can also remove `src_test`

File: sci-mathematics/nestedsums/nestedsums-1.5.2-r1.ebuild [hunk 2]
```diff
@@ -35,11 +37,12 @@ src_test() {
 
 src_install() {
 	emake DESTDIR="${D}" install
-	rm "${D}"/usr/$(get_libdir)/lib${PN}.la || die "cannot rm lib${PN}.la"
 	dodoc AUTHORS ChangeLog
```
Comment (SoapGentoo) [inline sci-mathematics/nestedsums/nestedsums-1.5.2-r1.ebuild:40]: `default` here?

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: sci-mathematics/nestedsums/nestedsums-1.5.2-r1.ebuild (hunk 1 above)
Comment (jsmolic) [reply sci-mathematics/nestedsums/nestedsums-1.5.2-r1.ebuild]: done

File: sci-mathematics/nestedsums/nestedsums-1.5.2-r1.ebuild (hunk 2 above)
Comment (jsmolic) [reply sci-mathematics/nestedsums/nestedsums-1.5.2-r1.ebuild]: done

**Pull request closed without merging**